micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/ProcessEngineFactory.java

```diff
@@ -50,7 +50,9 @@ public ProcessEngine processEngine(ProcessEngineConfiguration processEngineConfi
      * @throws IOException if a resource, i.e. a model, cannot be loaded.
      */
     private void deployProcessModels(ProcessEngine processEngine) throws IOException {
+        CamundaBpmVersion camundaBpmVersion = new CamundaBpmVersion();
```

Comment:
It would be nicer if CamundaBpmVersion was a bean, i.e. you annotate the class with '@Singleton` and then inject it to the method `processEngine`.